package.json

```diff
@@ -50,13 +50,8 @@
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "storybook": "start-storybook -p 9009 -s public",
     "build-storybook": "build-storybook -s public",
-    "sentry-upload-sourcemaps": "sentry-cli --auth-token $SENTRY_AUTH_TOKEN releases -o $SENTRY_ORG -p $SENTRY_PROJECT files $npm_package_version upload-sourcemaps ./build/static/js/"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged --allow-empty",
-      "pre-push": "tsc --noEmit --incremental"
```

Comment:
This doesn't work anymore?

Reply:
it's now inside `.husky` folder

Reply:
I saw it, just wondering if we could keep it in the json.

Reply:
I like it outside, personally not a big fan of putting everything into package.json